.env.example

```diff
@@ -39,3 +39,6 @@ USER_PASSWORD=
 
 # PDF generator (Grover/Puppeteer)
 GROVER_NO_SANDBOX=true
+
+# Google analytics tracking id
+TRACKING_ID: GTM-XXXXXXX 
```

Comment:
This needs to be in the format `FOO=bar` in case someone copies it without thinking

Reply:
Good spot